### ISSUE TITLE
Question about training on the custom dataset

### ISSUE THREAD

**Issue by IceIce1ce** [FelixCaae/CrowdSAM#12]
Hi author, thanks for your great code. In the config file of the CrowdHuman dataset, I found that the number of class is set to 1. In this case, if my custom dataset has multiple classes and my custom_dataset.json file has the same format as train_crowdhuman_10shot.json, the training code will have a problem with the training loss. To solve this problem, can the training loss of the coco be used for the custom dataset? Two training losses which I mention here is compute_loss_coco() and compoute_loss_crowdhuman() in crowdsam/loss.py.
